qml/DisplaySettings.qml

```diff
@@ -50,6 +50,39 @@ ColumnLayout {
                     onCurrentTextChanged: StyleManager.isDark = !currentIndex
                 }
 
+                CheckBox {
+                    id: alwaysOnTopCB
+                    text: "Enable Always-On-Top mode"
+                    checked: SettingsManager.alwaysOnTop
+                    Layout.columnSpan: 5
+                    Layout.fillWidth: true
+                    onCheckedChanged: {
+                        if (checked) {
+                            window.flags |= Qt.WindowStaysOnTopHint
+                        } else {
+                            window.flags &= ~Qt.WindowStaysOnTopHint
+                        }
+                        SettingsManager.alwaysOnTop = checked
+                    }
+                }
+
+                Label {
+                    text: "Opacity:"
+                }
+
+                ComboBox {
+                    id: opacityCB
+                    Layout.columnSpan: 4
+                    Layout.fillWidth: true
+                    currentIndex: SettingsManager.applicationOpacityIndex
+                    model: [100, 90, 80, 70]
```

Comment:
I'd prefer this to be a slider (if that is not a bunch of unnecessary work)

Reply:
It was discussed in private